### PR TITLE
Corrected PHP repo

### PR DESCRIPTION
**Pull request by toby-griffiths**
Fixes issue #296 by updating the PHP Repository name

### CODE REVIEW

File: ansible/roles/init/tasks/main.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 - name: Add ppa Repository
   become: yes
-  apt_repository: repo=ppa:ondrej/{{ php_ppa }}
```
Comment (InFog) [inline ansible/roles/init/tasks/main.yml]: Maybe you can change this on this variable's source.

Comment (toby-griffiths) [inline ansible/roles/init/tasks/main.yml]: @InFog  I'm happy to do this, but is it relevant?  Is it likely that this repo is likely to want configuring by the user/platform?

Comment (InFog) [inline ansible/roles/init/tasks/main.yml]: The problem is if we change it here the source of the variable will be useless and bugs may arise from that.

Comment (toby-griffiths) [inline ansible/roles/init/tasks/main.yml]: Ah.  That makes sense for now.

Comment (toby-griffiths) [inline ansible/roles/init/tasks/main.yml]: Updated the branch with this change now.

**Pull request merged**